src/PaymentGateways/PayPalCommerce/PayPalOrder.php

```diff
@@ -0,0 +1,144 @@
+<?php
+namespace Give\PaymentGateways\PayPalCommerce;
+
+use Give\Helpers\ArrayDataSet;
+use InvalidArgumentException;
+use stdClass;
+
+/**
+ * Class PayPalOrder
+ * @package Give\PaymentGateways\PayPalCommerce
+ *
+ * @since 2.8.0
+ */
+class PayPalOrder {
+	/**
+	 * Order Id.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $id;
+
+	/**
+	 * Order intent.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $intent;
+
+	/**
+	 * Order status.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $status;
+
+	/**
+	 * Order creation time.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $createTime;
+
+	/**
+	 * Order update time.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $updateTime;
+
+	/**
+	 * PayPal Order action links.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var string
+	 */
+	public $links;
+
+	/**
+	 * Payer information.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var stdClass
+	 */
+	public $payer;
+
+	/**
+	 * Order purchase unit details.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @var stdClass
```

Comment:
Do we know the shape of this? Can it be defined as a `PurchaseUnit` class?

Reply:
@JasonTheAdams I disagree. `purchase_unit` in order will be array of payment and in our case, there will be always one payment. To remove any confusion I decided to make it as private property and added a `payment` `public` property and removed `getPayment` function from this `PayPalOrder` class.

ref: PurchaseUnit